docs/self-hosted-appcircle/install-server/helm-chart/installation/docker-k8s-migration.md

```diff
@@ -0,0 +1,1519 @@
+
+   ```bash
+   kubectl get secret -n appcircle appcircle-server-minio-connection -ojsonpath='{.data.accessKey}' | base64 -d && \
+   echo && \
+   kubectl get secret -n appcircle appcircle-server-minio-connection -ojsonpath='{.data.secretKey}' | base64 -d
+   ```
+
+4. **Get the MinIO service name:**
+
+   ```bash
+   kubectl get services -n appcircle | grep minio
+   ```
+
+6. **Start port forwarding:**
+   ```bash
+   kubectl port-forward service/appcircle-server-minio 9000:9000 -n appcircle
+   ```
+
+6. **Install `rclone` tool:**
+
+   :::info
+   We recommended using `rclone` tool instead of `mc`.
+   :::
+
+   To install `rclone`, please check the [official rclone documentation](https://rclone.org/install/).
+
+7. **Add Rclone Configuration for Standalone Server:**
+
+   To configure Rclone for the standalone Appcircle server, follow these steps:
+
+   - Start the Rclone configuration process:
+
+     ```bash
+     rclone config
+     ```
+
+   - Use the following inputs during the configuration process:
+
+     ```plaintext
+     n                                # Create a new remote
+     name> ac-standalone              # Provide a descriptive name for the remote
+     Storage> 4                       # Select "Amazon S3 Compliant Storage Provider" (4)
+     provider> 7                      # Select "Minio Object Storage" (7)
+     env_auth> "false"                # Set environment authentication to "false"
+     access_key_id> <access_key>      # Enter the standalone Appcircle server's access key
+     secret_access_key> <secret_key>  # Enter the standalone Appcircle server's secret access key
+     region>                          # Leave this empty
+     endpoint>                        # Provide the standalone Appcircle server's IP and MinIO port. Example: http://192.168.1.220:9040
+     location_constraint:             # Leave this empty
+     acl:                             # Leave this empty
+     server_side_encryption:          # Leave this empty
+     sse_kms_key_id:                  # Leave this empty
+     Edit advanced config? (y/n): n   # Skip advanced configuration
+     ```
+
+8. **Add Rclone config for Kubernetes Appcircle server:**
+
+   To configure `rclone` for the Kubernetes Appcircle server, follow these steps:
+
+   - Start the `rclone` configuration process:
+
+     ```bash
+     rclone config
+     ```
+
+   - Use the following inputs during the configuration process:
+
+     ```plaintext
+     n                                # Create a new remote
+     name> ac-k8s                     # Provide a descriptive name for the remote
+     Storage> 4                       # Select "Amazon S3 Compliant Storage Provider" (4)
+     provider> 7                      # Select "Minio Object Storage" (7)
+     env_auth> "false"                # Set environment authentication to "false"
+     access_key_id> <access_key>      # Enter the standalone Appcircle server's access key
```

Comment:
- [ ] This section has "standalone" words in the comments that must be replaced with "K8s" side